wgpu-types/src/lib.rs

```diff
@@ -542,6 +542,15 @@ bitflags::bitflags! {
         ///
         /// This is a native only feature.
         const MULTIVIEW = 1 << 40;
+        /// Enables normalized `16-bit` texture formats.
```

Comment:
we need to document here which usages are guaranteed to be supported on these formats